cpp/ql/test/library-tests/dataflow/models-as-data/testModels.qll

```diff
@@ -16,7 +16,7 @@ private class TestSources extends SourceModelCsv {
         ";;false;remoteMadSourceIndirectArg0;;;Argument[*0];remote",
         ";;false;remoteMadSourceIndirectArg1;;;Argument[*1];remote",
         ";;false;remoteMadSourceVar;;;;remote",
-        ";;false;remoteMadSourceVarIndirect;;;*;remote",
+        ";;false;remoteMadSourceVarIndirect;;;*;remote", // not correctly expressed
```

Comment:
So here we want to specify that the first indirection of the global variable `remoteMadSourceVarIndirect` is a flow source, right? And the problem is that we have to use the `ext` column as a container for the indirection? How about we simply modify the case for global variables that I added in `interpretElement0` [here](https://github.com/github/codeql/pull/15371/commits/3719e10ce875b6ae4ece2347286520ded639cb3a)? Then we can include the stars in the name directly there

Reply:
That should work, but I think it will require https://github.com/github/codeql/pull/15977 (and thus another round of arguing our case on there).  I'll prototype it first and make sure it actually makes sense.

Reply:
OK, I agree its time to start carving off some follow-up work so that we can get the bulk of this merged.  I've created an issue for it.